app/controllers/snacks_controller.rb

```diff
@@ -0,0 +1,9 @@
+class SnacksController < ApplicationController
+
+  def show
+    @snack = Snack.find(params[:id])
+    #ran out of time due to back and forth emailing with landlord
+    # would have gotten @snack.machines sent the view at the very least
```

Comment:
I don't think you need to send @ snack.machines to the view since if you just send @ snack to the view, you have access to call .machines right off of that instance variable in the view.